_episodes/30.md

```diff
@@ -0,0 +1,244 @@
+---
+layout: episode
+title:  "30: Trino and dbt, a hot data mesh"
+date: 2021-11-17
+tags: dbt architecture sql infrastructure datamesh
+youtube_id: ""
+wistia_id: ""
+sections: 
+   - title: "Concept of the week"
+     desc: "Trino and dbt, a hot data mesh"
+     time: 0
+   - title: "PR of the week"
+     desc: "Partitioned table tests and fixed PR 9757"
+     time: 0
+   - title: "Question of the week"
+     desc: "What’s the difference between location and external_location?"
+     time: 0
+---
+
+## Guests
+
+ * José Cabeda, Data Engineer at [TalkDesk](https://www.talkdesk.com)
```

Comment:
Be consistent with either `TalkDesk` or `Talkdesk`. I think it's supposed to be the latter.